### ISSUE TITLE
Train a classifier, freeze the tree and retrain as a regressor

### ISSUE THREAD

**Issue by bangeneticalgorithms** [catboost/catboost#1334]
Problem: I would like to train a generate a CatbooostClassifier model and train it with loss_functions CrossEntropy and then freeze the trees, then train it as a Regressor with a loss function like RMSE. I understand right now we can't do this with to_classifier,to_regressor functions because the loss functions should match a Regressor object. 

So is there a work around for this? Can I save a model in a json format and edit some parameters and then reload it as a regressor?
catboost version:0.23.1
Operating System: Ubuntu 20.04
CPU: Intel Xeon
# GPU:

**Comment by kizill** [comment]
I don't understand what do you want - classification models predict values before sigmoid, so there is no way to simply train regression model on top of classifier model raw outputs or vice versa. But you could use one model predictions as baseline for next model:
```python
baseline_train = Pool(
    X_train, y_train,
    cat_features=categorical_features_indices,
    baseline=regression_model.predict(X_train)
)
baseline_eval = Pool(
    X_validation, y_validation,
    cat_features=categorical_features_indices,
    baseline=regression_model.predict(X_validation)
)
model = CatBoostClassifier(
    custom_loss=['Accuracy'],
    random_seed=42,
    logging_level='Silent'
)
model.fit(
    baseline_train,
    eval_set=baseline_eval,
    plot=True
)
```

**Comment by kizill** [comment]
Closing due to inactivity. Feel free to reopen this issue if you still have questions :)
